mappings/net/minecraft/entity/boss/dragon/phase/HoldingPatternPhase.mapping

```diff
@@ -1,5 +1,9 @@
 CLASS net/minecraft/class_1517 net/minecraft/entity/boss/dragon/phase/HoldingPatternPhase
 	FIELD field_18121 PLAYERS_IN_RANGE_PREDICATE Lnet/minecraft/class_4051;
-	FIELD field_7045 target Lnet/minecraft/class_243;
+	FIELD field_7043 path Lnet/minecraft/class_11;
+	FIELD field_7044 shouldFindNewPath Z
+	FIELD field_7045 pathTarget Lnet/minecraft/class_243;
+	METHOD method_6841 serverTickLogicWhenInRange ()V
```

Comment:
Can this tick on the client? If not, then
```suggestion
	METHOD method_6841 tickWhenInRange ()V
```

Reply:
its only server side.

![image](https://user-images.githubusercontent.com/13246889/111373705-7c446300-8659-11eb-87ef-859bfff3ef44.png)

it looks like mojang just move this code out into another method for clarity

Reply:
```suggestion
	METHOD method_6841 tickInRange ()V
```
That `serverTick` method most likely should be renamed to `tick` at some point.